db/seeds.rb

```diff
@@ -121,7 +121,7 @@
   user.sports = Sport.all.sample(i)
   e = rand(1..7)
   user.weekdays = Weekday.all.sample(e)
-  user.gym = Gym.all.sample
```

Comment:
I removed this part where a random gym was assigned to us admin users. For our purposes it might be better that Ida, Stella, Lydia, Kang are always at the same gym when a new seed is generated.